paraview/CMakeLists.txt

```diff
@@ -46,11 +46,15 @@ paraview_plugin_build(
     "${CMAKE_INSTALL_BINDIR}/plugins"
   LIBRARY_DESTINATION
     "${CMAKE_INSTALL_BINDIR}/plugins"
+  PLUGINS_FILE_NAME
+    "ttk.plugins.xml"
+  DELAYED_LOAD
+    TopologyToolKit
   )
 
 install(
   TARGETS
     TTKPVTargets
   DESTINATION
     "${TTK_INSTALL_PLUGIN_DIR}"
-  )
\ No newline at end of file
+  )
```

Comment:
This change is not needed but a good practices overall, let me know if I should keep the change or not.